sopel_8ball/plugin.py

```diff
@@ -27,7 +27,7 @@ def configure(settings: Config) -> None:
     managers.manager.configure(settings)
 
 
-@plugin.command('8ball')
+@plugin.commands('8ball', '8')
```

Comment:
Make it 8b and we have a deal.

```suggestion
@plugin.commands('8ball', '8b')
```